matplotlib-tutorial.ipynb

```diff
@@ -89,6 +89,7 @@
    },
```

Comment:
<p>You typod `encourrage`. It should be `encourage`.</p>

 _Reply via <a href='https://app.reviewnb.com/sebinsua/test-review-notebooks/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>